pkg/cli/job/list.go

```diff
@@ -104,11 +106,29 @@ func ListJobs(ctx context.Context) error {
 		return err
 	}
 
-	if len(jobs.Items) == 0 {
+	// define the filter callback function based on different flags
+	filterFunc := func(job v1alpha1.Job) bool {
+		// filter by QueueName if specified
+		if listJobFlags.QueueName != "" && listJobFlags.QueueName != job.Spec.Queue {
```

Comment:
vcjob has a lable associated with a queue，what about using label selector here？

Reply:
I think using job.Spec.Queue is a better choice

Reply:
And I didn't find any queue-related labels in the job. Only `QueueNameKey = "volcano.sh/queue-name"` but it will only be marked on the pod.

Reply:
My mistake,  we can use `job.Spec.Queue`: )

Reply:
ok :)